<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>orderslist</class>
 <widget class="QWidget" name="orderslist">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Orders</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>370</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>تایید</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnStartPreparing">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>380</y>
     <width>91</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>شروع آماده سازی</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreparing">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>در حال آماده سازی</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelivered">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>380</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="toolTip">
    <string>ارسال شد</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/images/orderlistbg.jpg);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="backbtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>31</width>
     <height>29</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:2px;

background-color: rgb(255, 39, 79);
border:0px solid;
color:black;
font: 900 9pt &quot;IRANYekanFN&quot;;
border-radius:5px;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>30</y>
     <width>640</width>
     <height>290</height>
    </rect>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 250, 93);
border-radius:8px;
color:black;</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>btnConfirm</zorder>
  <zorder>btnStartPreparing</zorder>
  <zorder>btnPreparing</zorder>
  <zorder>btnDelivered</zorder>
  <zorder>backbtn</zorder>
  <zorder>tableWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
